src/module.cc

```diff
@@ -1,7 +1,7 @@
 #include "node_report.h"
 #include <nan.h>
 
-namespace nodereport {
+namespace node-report {
```

Comment:
I think every change in this file pretty much, except maybe some comments, can be reverted as unnecessary (and won't compile).